mteb/tasks/Classification/multilingual/ScalaClassification.py

```diff
@@ -1,167 +1,56 @@
-    )
-
-    @property
-    def metadata_dict(self) -> dict[str, str]:
-        metadata_dict = super().metadata_dict
-        metadata_dict["n_experiments"] = 10
-        metadata_dict["samples_per_label"] = 32
-        return metadata_dict
-
-    def dataset_transform(self):
-        # convert label to a 0/1 label
-        labels = self.dataset["train"]["label"]  # type: ignore
-        lab2idx = {lab: idx for idx, lab in enumerate(set(labels))}
-        self.dataset = self.dataset.map(
-            lambda x: {"label": lab2idx[x["label"]]}, remove_columns=["label"]
-        )
+_LANGS = {
+    "Danish": ["dan-Arab"],
+    "Norwegian_b": ["nob-Hans"],
+    "Norwegian_n": ["nno-Latn"],
+    "Swedish": ["swe-Latn"],
+}
 
 
-class ScalaSvClassification(AbsTaskClassification):
+class MultiScalaClassification(AbsTaskClassification, MultilingualTask):
     metadata = TaskMetadata(
-        name="ScalaSvClassification",
-        description="A Swedish dataset for linguistic acceptability classification",
+        name="MultiScalaClassification",
+        description="""ScaLa - multilingual version of nordic languages dataset for linguistic acceptability classification.
+        Published as part of 'ScandEval: A Benchmark for Scandinavian Natural Language Processing'""",
         reference="https://aclanthology.org/2023.nodalida-1.20/",
         dataset={
-            "path": "mteb/scala_sv_classification",
-            "revision": "aded78beae37445bf3917102d0b332049cfc8c99",
+            "path": "mteb/multilingual-scala-classification",
+            "revision": "ec85bb6c69679ed15ac66c0bf6e180bf563eb137",
         },
         type="Classification",
         category="s2s",
         eval_splits=["test"],
-        eval_langs=["swe-Latn"],
+        eval_langs=_LANGS,
         main_score="accuracy",
         date=None,
-        form=None,
+        form=["written"],
         domains=None,
-        task_subtypes=None,
+        task_subtypes=["Linguistic acceptability"],
         license=None,
-        socioeconomic_status=None,
-        annotations_creators=None,
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
         dialect=None,
-        text_creation=None,
-        bibtex_citation=None,
+        text_creation="created",
+        bibtex_citation="""
+        @inproceedings{nielsen-2023-scandeval,
+            title = "{S}cand{E}val: A Benchmark for {S}candinavian Natural Language Processing",
+            author = "Nielsen, Dan",
+            editor = {Alum{\"a}e, Tanel  and
+            Fishel, Mark},
+            booktitle = "Proceedings of the 24th Nordic Conference on Computational Linguistics (NoDaLiDa)",
+            month = may,
+            year = "2023",
+            address = "T{\'o}rshavn, Faroe Islands",
+            publisher = "University of Tartu Library",
+            url = "https://aclanthology.org/2023.nodalida-1.20",
+            pages = "185--201",
+        }""",
         n_samples={"test": 1024},
```

Comment:
should be larger I assume this is only for 1

Reply:
hmm 🤔 I thought this is per single dataset, and here's easy cause they're consistent, or should it be `len(_LANGS)*1024`?

Reply:
It should be `len(_LANGS)*1024`